main.go

```diff
@@ -76,12 +77,26 @@ func BubbleSort(items [][]string) {
 			}
 		}
 	}
-
 }
-func LoadData() [][]string {
-	r, _ := os.Open("data/commits.csv")
+
+func LoadData() (records [][]string, err error) {
+	r, err1 := os.Open("data/commits.csv")
```

Comment:
You can add `defer r.Close()` right after you assign it because the function is called once the caller goes out of scope.  It isn't actually run when declared.

Reply:
k